react/features/prejoin/components/native/Prejoin.tsx

```diff
@@ -149,14 +163,12 @@ const Prejoin: React.FC<IPrejoinProps> = ({ navigation }: IPrejoinProps) => {
                 }
                 <Button
                     accessibilityLabel = 'prejoin.joinMeeting'
-                    disabled = { joinButtonDisabled }
                     labelKey = 'prejoin.joinMeeting'
-                    onClick = { isJoining ? undefined : onJoin }
+                    onClick = { isJoining ? undefined : maybeJoin }
                     style = { styles.joinButton }
                     type = { PRIMARY } />
                 <Button
                     accessibilityLabel = 'prejoin.joinMeetingInLowBandwidthMode'
-                    disabled = { joinButtonDisabled }
                     labelKey = 'prejoin.joinMeetingInLowBandwidthMode'
                     onClick = { onJoinLowBandwidth }
```

Comment:
Come to think of it, we should also disable this one in we are in the process of joining...